SDK/simVis/ElevationQueryProxy.h

```diff
@@ -137,6 +137,9 @@ class SDKVIS_EXPORT ElevationQueryProxy
   osg::observer_ptr<const osgEarth::Map> map_;
   osg::observer_ptr<osg::Group> scene_;
 
+  osgEarth::AsyncElevationSampler* asyncSampler_;
+  osgEarth::ElevationPool::WorkingSet workingSet_;
```

Comment:
Getting a build error without `#include "osgEarth/ElevationQuery"` in this header.  I can fix afterwards though, no problem.